src/chapters/classes-and-objects/project.rst

```diff
@@ -0,0 +1,127 @@
+Project: Classes and Objects
+==============================
+
+Now that we've got our robots up and running, we want to select a few of them
```

Comment:
the exercises and projects here are good - the only thing is this intro is a bit misleading. I was looking for a tie-in to the robot objects made in the exercises. Can you modify this to not reference them and/or state explicitly that the students will be creating cleaning robot objects made from a different class?

Reply:
Rephrased the introductory sentence to break the link to the exercise robots.